### PR TITLE
Handle stack traces with unknown contracts

### PR DESCRIPTION
**Pull request by benber86**
#57

Recurse when the contract is unknown and fill in the missing stack frames with the unknown contract's address.

### CODE REVIEW

File: boa/vyper/contract.py [hunk 1]
```diff
@@ -611,11 +611,19 @@ def handle_error(self, computation):
...
             raise strip_internal_frames(b) from None
 
-    def vyper_stack_trace(self, computation):
-        ret = StackTrace([ErrorDetail.from_computation(self, computation)])
-        error_detail = self.find_error_meta(computation.code)
-        if error_detail not in EXTERNAL_CALL_ERRORS:
-            return ret
+    def vyper_stack_trace(self, computation, unknown=False):
+        if unknown:
+            ret = StackTrace(
```
Comment (charles-cooper) [inline boa/vyper/contract.py:616]: i think it still needs to recurse here.

**Review by charles-cooper — COMMENTED** [review]
probably a cleaner approach is not to change `vyper_stack_trace` but add a helper (non-method) function, `trace_for_unknown_contract`

File: boa/vyper/contract.py (hunk 1 above)
Comment (benber86) [reply boa/vyper/contract.py:616]: it's not a return statement? it will still recurse below as long as there's a child computation

**Comment by benber86** [comment]
> probably a cleaner approach is not to change `vyper_stack_trace` but add a helper (non-method) function, `trace_for_unknown_contract`

indeed, that flag is a bit ugly. will do

**Review by charles-cooper — APPROVED** [review]

**Pull request merged**